### DISCUSSION TITLE
Execute on Paste not Working

### DISCUSSION THREAD

**Discussion by big-edd** [Eugeny/tabby#7287] [Q&A] [answered]
Executing commands as they are pasted, is something that I use/need, but this does not seem to work. 
The option for "Bracketed paste" is disabled, and the shells being used (PowerShell and Cygwin/bash) definitely allow for this when used outside of Tabby. 
Would there be another relevant setting that I might have missed, so that this can work? 

Note: Have noticed that when pasting multiple lines of commands, all will run except the last one.

**Comment by Eugeny** [comment, marked as answer]
Tabby automatically trims whitespace at the end of the pasted text to avoid accidental execution - this will become optional in the next release

**Comment by lawweiliang** [comment]
@big-edd 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
